webapp/backend/src/handlers/api/videos.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response, Router } from "express";
+import fs from "fs";
+import allVideos from "../../videoData";
+import path from "path";
+
+// Initialise router object
+const demoVideos = Router();
```

Comment:
let's keep the name of this variable as `routes` to be consistent with the naming in `index.ts` file

Reply:
> let's keep the name of this variable as `routes` to be consistent with the naming in `index.ts` file

I thought the names had to be different... or do you mean something like `demoVideoRoutes` or just `routes`?

Reply:
why do the names have to be different? the return of the Router() object is not specific to the demo routes is it? isn't it just a route object?

Reply:
Sure, it is just a route object. I will make the naming consistent as suggested. My response was a thought I had about routes. Thanks for clarifying.